cogs/qrcode.py

```diff
@@ -0,0 +1,26 @@
+import discord
+from discord.ext import commands
+import os
+from MyQR import myqr
+
+class qr_code(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def qr(self, ctx, sub1):
+        generate(str(ctx.author.id), sub1)
+        with open(os.path.join(os.getcwd(), "src") + "/" + str(ctx.author.id)+".png", 'rb') as f:
+            picture = discord.File(f)
+            await ctx.channel.send(file=picture)
+        
+    
+
+def generate(name, subject):
+    global qr, photo, filename, save_dir, timestampStr1,type1
+    version, level, qr = myqr.run( subject, version=1, level='H', colorized=True, contrast=1.0, brightness=1.0, save_name=name+".png", save_dir=os.path.join(os.getcwd(), "src"))
```

Comment:
This will hurt the repl.it storage. It will be great if the image is not saved or deleted after message sent

Reply:
consider using Imgur API [https://api.imgur.com/endpoints/image/](https://api.imgur.com/endpoints/image/)